### DISCUSSION TITLE
ESM build appears to be missing files

### DISCUSSION THREAD

**Discussion by emyarod** [codesandbox/codesandbox-client#4273] [General] [answered]
hello, some time during the last couple of days we started experiencing issues with ESM builds. Here is a link to our template:

https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

when looking into the error log, we see `Unable to open 'mixin.js': File not found (file:///sandbox/node_modules/carbon-components/es/globals/js/misc/mixin.js)`, but this file seems to output correctly in local builds. I'm not sure how to proceed with troubleshooting this but at the moment all of our templates and examples are affected by this build problem, so any help or advice would be greatly appreciated!

**Comment by garethx** [comment, marked as answer]
We've made some changes and the above example is now working. Let us know if your other examples still have issues.

**Reply by emyarod** [reply]
seems to be working fine again! thank you
